README.md

```diff
@@ -59,13 +59,13 @@ gem "launchy"
 gem "rack"
 ```
 
-A few of these will likely be new to you. You'll have plenty of time to familiarize yourself, but at a high level.
+A few of these will likely be new to you. You'll have plenty of time to familiarize yourself, but at a high level:
```

Comment:
We don't officially explain Gemfiles or bundle in Mod 1; might want to include a short reminder as to what they do.